packages/sdk/src/reads/getBalancerMinterRewards.ts

```diff
@@ -0,0 +1,30 @@
+import { type ReadContractParameters, readContractParameters } from "../utils/viem.js";
+import type { Address, PublicClient } from "viem";
+
+const abi = {
+  inputs: [{ internalType: "address", name: "gauge", type: "address" }],
+  name: "mint",
+  outputs: [{ internalType: "uint256", name: "", type: "uint256" }],
+  stateMutability: "nonpayable",
+  type: "function",
+} as const;
+
+export async function getBalancerMinterRewards(
+  client: PublicClient,
+  args: ReadContractParameters<{
+    beneficient: Address;
```

Comment:
It should technically be `beneficiary` I believe, unless of course `beneficient` is already used elsewhere.